docker-compose.yml

```
@@ -79,6 +79,13 @@ services:
       ELASTIC_PASSWORD: ${ELASTIC_PASSWORD}
       ELASTICSEARCH_HOST_PORT: ${ELASTICSEARCH_HOST}:${ELASTICSEARCH_PORT}
       LS_JAVA_OPTS: "-Xmx${LOGSTASH_HEAP} -Xms${LOGSTASH_HEAP}"
+    secrets:
+      - source: elastic.ca
+        target: /certs/ca.crt
+    ports:
+      - "5044:5044/tcp"
+      - "5044:5044/udp"
+      - "9600:9600"
```

Enable for all protocols at once for the sake of simplicity.
```suggestion
    ports:
      - "5044:5044"
      - "9600:9600"
```